components/typography.js

```diff
@@ -40,17 +47,29 @@ export function Headline({ children, alpha, style, level, ...props }) {
   )
 }
 
-export function BodyText({ children, alpha, style, ...props }) {
-  const textColor = color(theme.colors.text).alpha(alpha).rgb().string()
+export function BodyText({
+  children,
+  color = theme.colors.text,
+  alpha = 1,
```

Comment:
minor: the idiomatic react way to set default prop values is via Component.defaultProps

Reply:
I'm happy to change it. However, worth noting that the `defaultProps` API is likely going to be deprecated in future versions. https://github.com/reactjs/rfcs/pull/107.